### ISSUE TITLE
commented code

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1209]
should not report the full thing as commented code, first and last is not ABAP

```abap
*    Description part 2:
*
*    IF io_xml->i18n_params( )-serialize_master_lang_only = abap_true.
*      ls_lang_sel-low = mv_language.
*      ls_lang_sel-sign = 'I'.
*      ls_lang_sel-option = 'EQ'.
*    ENDIF.
*
*    SELECT * FROM cls_type_groupt INTO TABLE ls_otgr-texts
*      WHERE name = ms_item-obj_name
*        AND activation_state = 'A'
*        AND langu in lt_lang_sel.
*
*  Description ideas end.
```

![image](https://user-images.githubusercontent.com/5888506/90913431-8d3b3780-e3dc-11ea-9e61-cc7ea6b89a39.png)
